src/clj/web/api.clj

```diff
@@ -106,8 +113,8 @@
 
 (defroutes routes
   (wrap-routes private-routes wrap-anti-forgery)
-  (wrap-routes public-CSRF-routes wrap-anti-forgery)
-  public-routes)
+  public-routes
```

Comment:
Any reason for this?

Reply:
The `public-CSRF-routes` has this route handler `(GET "/*" [] pages/index-page)`, which blocks the `GET` in `public-routes`. You still want it with the CSRF token, so pulling it out doesn't make a ton of sense.

Reply:
That makes sense.